changelogs/fragments/drop_db_with_force.yml

```diff
@@ -0,0 +1,3 @@
+postgresql_db name=mydb state=absent fails to remove mydb database if it has active connections.
+
+Added a force option (boolean) to remove active connections first and then remove the database.
```

Comment:
```suggestion
minor_changes:
- postgresql_db - Add the ``force`` boolean option to drop active connections first and then remove the database (https://github.com/ansible-collections/community.postgresql/issues/109).
```

Reply:
this should fix the sanity errors

Reply:
for details about the fragments, refer to https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment